content/file-and-directory-naming/java/_index.md

```diff
@@ -12,10 +12,11 @@ title: File and directory naming in Java
 ---
 
 ## How should I name my Java files and directories?
-When you think about how to name files and directories in Java, you should know that different Java programmers can have different styles and approaches to the way they program. When naming stuff you must use Java standard conventions that are readable and meaningful to yourself and other programmers.
 
+When you think about how to name files and directories in Java, you should know that different Java programmers can have different styles and approaches to the way they program. When naming stuff you must use Java standard conventions that are readable and meaningful to yourself and other programmers.
 
 ### Directories
+
 Directory names in Java should be in lower case which means that you only use small letters. Eg `src` or `main`
```

Comment:
It would be worth saying that if you are using a tool like Gradle to set up your project structure, then just go with the conventions it sets up.